assets/src/graph/cdServices.graphql

```diff
@@ -165,8 +165,8 @@ query ServiceDeployments(
   }
 }
 
-query ServiceDeploymentsTiny {
-  serviceDeployments(first: 100) {
+query ServiceDeploymentsTiny($clusterId: ID!) {
```

Comment:
Is this only used in the selector?  We can type the `$clusterId` arg as `ID` and allow it to still select all services for other usecases if need be (by omitting or null-ing the $clusterId arg).

Reply:
yes, it is only used in the selector, but I agree, allowing it to be null is a good idea. I'll change that.